pennylane/devices/default_mixed.py

```diff
@@ -181,9 +224,17 @@ def _apply_channel(self, kraus, wires):
         kraus_dagger = [self._conj(self._transpose(k)) for k in kraus]
 
         # Changes tensor shape
-        kraus_shape = [len(kraus)] + [2] * num_ch_wires * 2
-        kraus = self._cast(self._reshape(kraus, kraus_shape), dtype=self.C_DTYPE)
-        kraus_dagger = self._cast(self._reshape(kraus_dagger, kraus_shape), dtype=self.C_DTYPE)
+        if kraus[0].shape[0] == kraus[0].shape[1]:
+            kraus_shape = [len(kraus)] + [2] * num_ch_wires * 2
+            kraus = self._cast(self._reshape(kraus, kraus_shape), dtype=self.C_DTYPE)
+            kraus_dagger = self._cast(self._reshape(kraus_dagger, kraus_shape), dtype=self.C_DTYPE)
+        elif (kraus[0].shape == (1, 2)) and (num_ch_wires == 1):
+            kraus_shape = [len(kraus)] + list(kraus[0].shape)
+            kraus = self._cast(self._reshape(kraus, kraus_shape), dtype=self.C_DTYPE)
+            kraus_dagger_shape = [len(kraus)] + list(kraus[0].shape)[::-1] * num_ch_wires
+            kraus_dagger = self._cast(
+                self._reshape(kraus_dagger, kraus_dagger_shape), dtype=self.C_DTYPE
+            )
```

Comment:
Is this change primarily to support passing kraus operators of the partial trace? I might tag @ixfoduap for verification 🙂

Reply:
It only accepted Kraus operators of size nxn (e.g. 2x2, 4x4) and the one for partial traces have shape (1x2). I was unable to change the code for accepting every type of shapes, therefore I added the needed case. But it would be great to change it.